includes/class-plugin.php

```diff
@@ -699,4 +702,11 @@ public function on_deactivation( $plugin ) {
 
         ob_end_clean();
     }
+
+    public function action_link( $action ) {
```

Comment:
@naxvog: What's this for?

Reply:
The method should be part of another PR will revert it (for now 😉)